hw2/05-form.html

```diff
@@ -7,13 +7,96 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
+  <body class="bg-dark">
+    <main class="container bg-light w-50">
+      <form id="form">
+        <h1 class="mt-5 p-2 text-center">Exercise 05 - Form</h1>
+        <div class="m-2">
+          <label for="name" class="form-group fw-bold">Full Name:</label>
+          <input id="name" class="form-control mt-2" type="text" />
+        </div>
+        <div class="m-2">
+          <label for="email" class="form-group fw-bold">Email:</label>
+          <input id="email" class="form-control mt-2" type="email" />
+        </div>
+        <div class="m-2">
+          <label for="options" class="form-group fw-bold"
+            >Registration Status:</label
+          >
+          <select required class="form-control form-select" id="options">
+            <option value="" selected>Choose an Option</option>
+            <option value="Registered">Registered</option>
+            <option value="Not-Registered">Not-Registered</option>
+            <option value="Waitlisted">Waitlisted</option>
+          </select>
+        </div>
+        <div class="m-2">
+          <label for="checkboxes" class="checkboxes form-group fw-bold"
```

Comment:
fieldset element of html can remove WAVE tool alert here. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset